Features & Bug Fix

17hoehbr opened the pull request:
There is a bug in the website where for some reason the html link to C SPAN does not follow the same convention as the rest of the links on the site so I added a separate line to make it go back and fetch any channels that have the same bug.

I also added a tvg-id section so that the generated m3u can be automatically mapped in xteve. Then I created seperate json files that users can modify to change the tvg-id for any channel to whatever matches their xmltv file.

Finally I also added another json for channel category mapping. I included a pre-made category map but again users can change it up to their liking by simply modifying the channel_category.json file.

17hoehbr commented:
Oh and I also deleted the included ustvgo.m3u8 file because it did not seem to work when I used it anyway. I suppose I could upload my newly generated one though.

interlark commented on download.py:
```diff
@@ -199,7 +208,7 @@ def check_gecko_driver():
         file.write('#EXTM3U\n\n')
         for name, url in video_links:
-            file.write('#EXTINF:-1,' + name + '\n')
+            file.write('#EXTINF:-1 tvg-id="' + channel_id_override[name] + '" ' + 'group-title="' + channel_categories[name] + '", ' + name + '\n')
```
You better get dictionary values with ```.get(key, default_value)``` to avoid ```KeyError```'s in case ```ustvgo``` would change their channel names.

interlark reviewed (changes requested):
I think I like it. Also, please, add your generates ```ustvgo.m3u8``` so people could use it without generating their own playlist. With existing playlist the only thing they need is to update the key with [ustvgo.m3u8](https://github.com/interlark/ustvgo_downloader/blob/master/update.py).

17hoehbr commented:
Added the new ustvgo.m3u8 file.

I also noticed that the script was not properly detecting VPN restricted content causing it to time out instead of just skipping. It seems that they changed the text on the VPN required screen so I updated the script to accommodate.

Then I implemented the dictionary.get method to avoid potential KeyErrors and added further code to prevent an error if either json file were missing.

17hoehbr commented:
Oh and I cleaned up the EXTINF file.write section by using string substitution instead of concatenation
